internal/pkg/interaction/prompt.go

```diff
@@ -108,27 +108,12 @@ func (p *Prompt) Ask(q *Question) (result string, ok bool) {
 	return result, p.handleError(err)
 }
 
-func (p *Prompt) Select(s *Select) (result string, ok bool) {
-	// Ask only in the interactive terminal
-	if !p.Interactive {
-		return "", false
-	}
-
-	// Print description
-	if len(s.Description) > 0 {
-		p.Printf("\n%s\n", s.Description)
-	} else {
-		p.Printf("\n")
-	}
-
-	// Validator
-	opts := p.getOpts()
-	if s.Validator != nil {
-		opts = append(opts, survey.WithValidator(s.Validator))
-	}
+func (p *Prompt) Select(s *Select) (value string, ok bool) {
+	return value, p.doSelect(s, &value)
+}
 
-	err := survey.AskOne(&survey.Select{Message: s.Label, Help: s.Help, Options: s.Options, Default: s.Default}, &result, opts...)
-	return result, p.handleError(err)
+func (p *Prompt) SelectIndex(s *Select) (index int, ok bool) {
```

Comment:
Mame metodu `Select`, ktora v konzole vykresli `select` a vrati vybranu hodnotu.
No potrebujem aj metodu `SelectIndex`, ktora vrati, ktora hodnota v poradi sa vybrala.

Je to preto, lebo ked vyberam komponentu, tak sa zobrazuje jej nazov, ... no ja potrebujem pre dalsiu pracu index z pola `components`, ... aby som ziskal celu strukturu `Component`, nie iba vygenerovany nazov.

Reply:
Kniznica, ktoru pouzivame automaticky vrati hodnotu -> ak `string`, ... a index ak je typ `int`.